lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -101,6 +101,35 @@ def merge_includes(hash1, hash2)
   end
 end
 
+def assert_klass_has_instance_method(klass, instance_method)
+  begin
+    klass.instance_method(instance_method)
+  rescue NameError => err
+    warn "#{klass} is missing the method our prepended code depends on: #{err}."
+    warn "Was the method removed or renamed upstream? See: #{__FILE__}"
```

Comment:
we might want to colorize this

Reply:
Why not just raise an exception?  If we can't patch the method, we have bigger problems.

Reply:
yeah, I grabbed it from core... we can do it here since we are fine with bumping the major version